### PR TITLE
Update readme with info on retroactively adding acts_as_list

### PR DESCRIPTION
**Pull request by tinderfields**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,31 @@ In `acts_as_list`, "higher" means further up the list (a lower `position`), and
 - `list_item.lower_item`
 - `list_item.lower_items` will return all the items below `list_item` in the list (ordered by the position, ascending)
 
+## Adding ActsAsList To An Existing Model
```
Comment (brendon) [inline README.md]: For consistency can you call it `acts_as_list` (use backticks)?

File: README.md [hunk 2]
```diff
@@ -70,6 +70,31 @@ In `acts_as_list`, "higher" means further up the list (a lower `position`), and
...
+As it stands ActsAsList requires position values to be set on the model before the instance methods above will work. Adding something like the below to your migration will set the default position. Change the parameters to order if you want a different initial ordering.
+
+```ruby
+class AddPositionToTodoItem < ActiveRecord::Migration
+  def change
+    add_column :todo_items, :position, :integer
+
```
Comment (brendon) [inline README.md]: You might run into the need to do this before operating on the class:

```ruby
TodoItem.reset_column_information
```

File: README.md [hunk 3]
```diff
@@ -70,6 +70,31 @@ In `acts_as_list`, "higher" means further up the list (a lower `position`), and
...
+      todo_item.update_columns(position: i + 1)
+    end
+  end
+end
+```
+
+If you are using the scope option things can get a bit more complicated. Let's say you have `acts_as_list scope: :todo_list`, you might instead need something like this:
+
+```ruby
+TodoList.find(TodoItem.group(:todo_list_id).pluck(:todo_list_id)).each do |todo_list|
```
Comment (brendon) [inline README.md]: You could also do: `TodoItem.distinct(:todo_list_id).pluck(:todo_list_id)`

File: README.md [hunk 4]
```diff
@@ -109,7 +134,7 @@ All versions `0.1.5` onwards require Rails 3.0.x and higher.
 2. Rails 4 compatibility and bye bye Rails 2! Older versions would of course continue to work with Rails 2, but there won't be any support on those.
 
 ## Contributing to `acts_as_list`
- 
+
```
Comment (brendon) [inline README.md]: Are you able to fix the whitespace differences as they distract from your changes. :) I really appreciate your work on `acts_as_list`! I just want to make sure everything stays tidy and concise :)

File: README.md (hunk 2 above)
Comment (tinderfields) [inline README.md]: I tested this and it doesn't seem to be required - I guess `update_columns` just writes raw css

File: README.md [hunk 5]
```diff
@@ -70,6 +70,31 @@ In `acts_as_list`, "higher" means further up the list (a lower `position`), and
 - `list_item.lower_item`
 - `list_item.lower_items` will return all the items below `list_item` in the list (ordered by the position, ascending)
 
+## Adding `acts_as_list` To An Existing Model
+As it stands ActsAsList requires position values to be set on the model before the instance methods above will work. Adding something like the below to your migration will set the default position. Change the parameters to order if you want a different initial ordering.
```
Comment (brendon) [inline README.md]: This line needs to be swapped to `acts_as_list` also :)

File: README.md (hunk 2 above)
Comment (brendon) [inline README.md]: Ah yep, that's true :)

**Comment by brendon** [comment]
Just that one tweak and I'll merge this for you :)

**Comment by brendon** [comment]
@tinderfields, do you want to finish off that one change and I can close and merge this :)

**Comment by brendon** [comment]
I've taken your update and created my own commit on master.

**Pull request closed without merging**